Automattic/jetpack
# Modules: Use the module slug in get_available_modules

#### Changes proposed in this Pull Request:
* The `Jetpack::get_available_modules` method uses the module file in the call to `Jetpack::get_module`. It looks like the other calls to `Jetpack::get_module` use the module slug, so let's do the same here.
* Add a description of the $module param to `Jetpack::get_module`.

#### Jetpack product discussion
* n/a

#### Does this pull request change what data or activity we track or use?
* no

#### Testing instructions:
1. Install this branch. Activate and connect Jetpack.
2. Navigate to `{your site's url}/wp-admin/admin.php?page=jetpack_modules`. Confirm that the modules are displayed.
